feat: fix hu-HU translations

peter-gy opened the pull request:
Tuning the already surprisingly good AI-translated strings from #203

runeb commented:
Thank you for your thorough review, @peter-gy! I have added a question for you about "Perspective(s)", otherwise ready to merge

peter-gy commented:
No worries, @runeb. Where should I see your question?

runeb commented:
> No worries, @runeb. Where should I see your question? I can't see any review comment or feedback on my commit.

My bad @peter-gy, I had not actually submitted it! 🙈 Should be visible now

peter-gy commented on locales/hu-HU/src/vision.ts:
```diff
@@ -58,12 +58,12 @@ export default removeUndefinedLocaleResources({
    * @see {@link https://www.sanity.io/docs/perspectives}
    */
-  'settings.perspective-label': 'Perspective',
+  'settings.perspective-label': 'Nézet',
```
Sure, this makes sense. We also say "Perspektíva" in Hungarian, so sticking with "Perspective" should be intuitive enough. I just pushed the changes.

peter-gy commented:
> > No worries, @runeb. Where should I see your question? I can't see any review comment or feedback on my commit.
> 
> My bad @peter-gy, I had not actually submitted it! 🙈 Should be visible now

All good, I just addressed your comments.
